_posts/2024-08-06-quarkus-and-leyden.adoc

```diff
@@ -0,0 +1,275 @@
+---
+layout: post
+title: 'Project Leyden'
+date: 2024-08-06
+tags: leyden
+synopsis: 'Project Leyden: exploring its potential for Quarkus users.'
+author: leydenquarkus
+---
+:imagesdir: /assets/images/posts/leyden
+
+You might have heard of https://openjdk.org/projects/leyden/[Project Leyden]: an initiative within the OpenJDK project with very ambitious goals.
+
+When casually namedropping "I'm looking into Project Leyden" with some fellow Java developers these days, I would usually spot some off guard thinking: "Leyden..?", to then offer a hasty save "Ah, right, the project to improve startup times".
+
+Which is not wrong, as indeed startup times improvements are one of its goals, yet we believe the other stated goals of the project offer even greater potential for our favourite platform and its users.
```

Comment:
```suggestion
Which is not wrong, as indeed startup times improvements are one of its goals, yet we believe the other stated goals of the project offer even greater potential for our favourite platform, Quarkus, and its users.
```

better to be explicit :)

Reply:
+1, I actually thought the same thing when reading.